docs/pages/getting-started/installation.md

```diff
@@ -28,6 +28,18 @@ docker run chainsafe/lodestar --help
 Docker is the recommended setup for Lodestar. Use our [Lodestar Quickstart scripts](https://github.com/ChainSafe/lodestar-quickstart) with Docker for detailed instructions.
 :::
 
+Speed improvements can be achieved during docker image build by relying on yarn [offline mirror](https://classic.yarnpkg.com/blog/2016/11/24/offline-mirror/).
```

Comment:
This does not belong here, the getting started/installation instructions are for users running Lodestar, not developers building the docker image locally